doc/man7/EVP_SIGNATURE-ED25519.pod

```diff
@@ -64,16 +120,24 @@ specified, then both Ed25519 and Ed448 are benchmarked.
 
 =head1 EXAMPLES
 
-To sign a message using a ED25519 or ED448 key:
+To sign a message using an ED25519 EVP_PKEY structure:
 
     void do_sign(EVP_PKEY *ed_key, unsigned char *msg, size_t msg_len)
     {
         size_t sig_len;
         unsigned char *sig = NULL;
         EVP_MD_CTX *md_ctx = EVP_MD_CTX_new();
 
-        EVP_DigestSignInit(md_ctx, NULL, NULL, NULL, ed_key);
-        /* Calculate the requires size for the signature by passing a NULL buffer */
+        const OSSL_PARAM params[] = {
+            OSSL_PARAM_utf8_string ("instance", "Ed25519ctx", 10),
+            OSSL_PARAM_octet_string("context-string", (unsigned char *)"A protocol defined context string", 24),
```

Comment:
You might want to assign this to a string first... The length of 24 doesnt seem to match this.

Reply:
hmm... not sure why I had 24 there rather than 33.  I've fixed it now.